packages/mds-provider/event-processor/processors/status-changes-processor.ts

```diff
@@ -66,19 +68,18 @@ function asStatusChange(entry: StatusChangesProcessorEntry): StatusChange {
     event_location: asPointFeature(telemetry),
     battery_pct: (telemetry && telemetry.charge) || null,
     associated_trip: trip_id || null,
-    recorded: now()
+    recorded,
+    sequence
   }
 }
 
-const isVehicleEventEntry = <Label>(
-  entry: LabeledStreamEntry<Label>
-): entry is LabeledStreamEntry<Label, VehicleEvent, 'event'> =>
+const isVehicleEventEntry = (
```

Comment:
I think a better name for this would be `isVehicleEventStreamEntry`

Reply:
It's now called `isStatusChangesProcessorStreamEntry` .. I simplified the type-guarding.